.github/workflows/workflow.yml

```diff
@@ -26,15 +26,6 @@ jobs:
           sudo apt-get autoremove
           sudo apt-get autoclean
           pip install pipenv
-      - name: Pipenv install requirements and check it can be locked
-        run: |
-          pipenv --clear
-          pipenv install --dev --skip-lock
-          pipenv run pip install -e .[all]
-          pipenv run pip install --no-deps file:plugins/aea-ledger-ethereum
-          pipenv run pip install --no-deps file:plugins/aea-ledger-cosmos
-          pipenv run pip install --no-deps file:plugins/aea-ledger-fetchai
-          pipenv lock
```

Comment:
The lock check is timing out. We have faced this issues before once and it was caused by the PYPI not working so I've skipped the check temporarily. https://github.com/valory-xyz/open-aea/issues/390

Reply:
alright I will do the same on my PR #381 then

Reply:
Please add back in. It has to pass. Otherwise we need to fix it first.